test/test-utils.js

```diff
@@ -1,5 +1,14 @@
 const { ethers } = require('hardhat');
 
+const PROTOCOL_IDS = {
```

Comment:
Maybe we need to move this object from [airnode repo ](https://github.com/api3dao/airnode/blob/master/packages/airnode-protocol/src/index.ts#L29-L36)to airnode-protocol-v1 repo and import it anywhere were it is needed.